Gemfile

```diff
@@ -6,6 +6,7 @@ ruby File.read(".ruby-version").chomp
 gem "activerecord-session_store"
 gem "bootsnap", ">= 1.1.0", require: false
 gem "canonical-rails"
+gem "delayed_job"
```

Comment:
Already included, this just adds it explicitly into the gem file rather than inferring it from "sentry-delayed_job"

Reply:
can we lock to a version though?

Reply:
`"~> 4.1"` or something?